proxy/core/src/main/resources/cloudstate-common.conf

```diff
@@ -3,6 +3,9 @@
 akka {
   actor {
     provider = cluster
+    allow-java-serialization = off
+    warn-about-java-serializer-usage = off
+    internal-dispatcher = akka.actor.default-dispatcher // FIXME consider removing this line
```

Comment:
@pvlugter @patriknw Probably best solved by a decision—given that we control all that runs in this, I thouht it important to keep number of threads down. If you both agree with this setting, then I can remove the comment.

Reply:
I think it's a valid choice for Cloudstate

Reply:
Thanks @patriknw!